### ISSUE TITLE
Issues I face using development branch

### ISSUE THREAD

**Issue by hisham2007** [CosmicMind/Material#537]
Thanks for great effort.

I am actively using development branch in porting my material Android app to IOS.
[https://play.google.com/store/apps/details?id=com.tabe3.news](https://play.google.com/store/apps/details?id=com.tabe3.news)

So Please let me use this thread to report issues I face . I will add comment when I face new issue or enhancement suggestion.

most of my issues are with tableview

1- RTL support specially in Toolbar. toolbar doesn't mirror automstically when I change language. Also toolbar has methods " toolbar.rightViews.append",  "toolbar.leftViews.append"  while I think it is better to rename them to " toolbar.startViews.append",  "toolbar.endViews.append"   where they append views in the right place according to current RTL status.

2- When I use card view as table view cell, I can't get its size to not hardcode row height

3- When I use card view as table view cell and give it some padding `contentView.layout(card).horizontally(left: 16, right: 16).center()`  the pulse effect happens to the cell behind card not the cars itself

4- When I use card view as table view cell, to overcome RTL support for now, I use `toolbar.titleLabel.textAlignment = .right
        toolbar.detailLabel.textAlignment = .right`  . using this code, it is displayed correctly. but for a strange reason, when I scroll down the table view and scroll up again, labels are displayed aligned left while it should be right.
5- When I use card view as table view cell, toolbar image is displayed centered stretched, I dont know how to put it in right and not stretched.

**Comment by adamdahan** [comment]
@danieldahan This is something that can be discussed within the community on Gitter. Please close this. 

@hisham2007 If you would like you can privately share your code with us and we can help you out but this isn't the forum for recommendations.  The comments you have left are awesome and we appreciate it but we would need to see your code before we can consider these issues.

Cheers

**Comment by danieldahan** [comment]
@hisham2007, @adamdahan is correct. We should bring this to the [Gitter](https://gitter.im/CosmicMind/Material) channel. There are some points you made that are interesting. Let's discuss them there and leave GitHub strictly for relevant issues. Thank you!

**Comment by adamdahan** [comment]
@hisham2007 Also please migrate to the Master branch - Material 2 has been released and development has been merged into master.

**Comment by hisham2007** [comment]
Ok I will move there. thanks for quick reply. wish you the best
